uptane-standard.md

```diff
@@ -807,19 +808,20 @@ Full verification MAY be performed by either primary or secondary ECUs. The proc
 
 If {{TAP-5}} is supported, a primary ECU SHALL download metadata and images following the rules specified in that TAP.  If {{TAP-5}} is not supported, the download should follow the {{TUF-spec}} and the metadata file renaming rules specified in {{metadata_filename_rules}}.
 
+Before starting full verification, the repository mapping metadata MUST be consulted, to determine where to download metadata from. This procedure assumes the basic Uptane case: there are only two repositories (Director and Image), and all image paths are required to be signed by both repositories. If a more complex repository layout is being used, refer to {{DEPLOY}} for guidance on how to determine where to download metadata from.
+
 In order to perform full verification, an ECU SHALL perform the following steps:
 
-1. Load the repository mapping metadata ({{repo_mapping_meta}}), and use the information therein to determine from where metadata should be downloaded.
-2. Load and verify the current time, or the most recent securely attested time.
-3. Download and check the Root metadata file from the Director repository, following the procedure in {{check_root}}.
-4. Download and check the Timestamp metadata file from the Director repository, following the procedure in {{check_timestamp}}.
-5. Download and check the Snapshot metadata file from the Director repository, following the procedure in {{check_snapshot}}.
-6. Download and check the Targets metadata file from the Director repository, following the procedure in {{check_targets}}.
-7. Download and check the Root metadata file from the Image repository, following the procedure in {{check_root}}.
-8. Download and check the Timestamp metadata file from the Image repository, following the procedure in {{check_timestamp}}.
-9. Download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
-10. Download and check the top-level Targets metadata file from the Image repository, following the procedure in {{check_targets}}.
-11. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
+1. Load and verify the current time, or the most recent securely attested time.
+1. Download and check the Root metadata file from the Director repository, following the procedure in {{check_root}}.
```

Comment:
```suggestion
2. Download and check the Root metadata file from the Director repository, following the procedure in {{check_root}}.
```

Reply:
I'd prefer to leave this as-is, because it's generally a good practice to avoid numbering lists in markdown sources. It's a bug avoidance strategy. The markdown renderer will renumber the list starting at 1 every time a new list or sublist starts, no matter what's there (and do it silently, with no warnings). Numbering the lists in the source can lead you to think that it will matter, and show up in the output. Keeping all the numbers at 1 in the source reminds you that you don't control the numbering.